src/main/java/com/powsybl/balances_adjustment/pevf/PevfExchangesXml.java

```diff
@@ -0,0 +1,323 @@
+ */
+public final class PevfExchangesXml {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PevfExchangesXml.class);
+
+    // Log messages
+    private static final String UNEXPECTED_TOKEN = "Unexpected token: {}";
+
+    // Metadata
+    private static final String ROOT_ELEMENT_NAME = "ReportingInformation_MarketDocument";
+    private static final String MRID_ELEMENT_NAME = "mRID";
+    private static final String REVISION_NUMBER_ELEMENT_NAME = "revisionNumber";
+    private static final String TYPE_ELEMENT_NAME = "type";
+    private static final String PROCESS_TYPE_ELEMENT_NAME = "process.processType";
+    private static final String SENDER_MARKET_PARTICIPANT_ELEMENT_NAME = "sender_MarketParticipant";
+    private static final String CODING_SCHEME_ELEMENT_NAME = "codingScheme";
+    private static final String MARKET_ROLE_ELEMENT_NAME = "marketRole.type";
+    private static final String RECEIVER_MARKET_PARTICIPANT_ELEMENT_NAME = "receiver_MarketParticipant";
+    private static final String CREATION_DATETIME_ELEMENT_NAME = "createdDateTime";
+    private static final String TIME_PERIOD_INTERVAL_ELEMENT_NAME = "time_Period.timeInterval";
+    private static final String START_ELEMENT_NAME = "start";
+    private static final String END_ELEMENT_NAME = "end";
+    private static final String DATASET_MARKET_DOCUMENT_ELEMENT_NAME = "dataset_MarketDocument";
+    private static final String DOC_STATUS_ELEMENT_NAME = "docStatus";
+    private static final String VALUE_ELEMENT_NAME = "value";
+    // TimeSeries
+    private static final String TIMESERIES_ELEMENT_NAME = "TimeSeries";
+    private static final String BUSINESS_TYPE_ELEMENT_NAME = "businessType";
+    private static final String PRODUCT_ELEMENT_NAME = "product";
+    private static final String IN_DOMAIN_ELEMENT_NAME = "in_Domain";
+    private static final String OUT_DOMAIN_ELEMENT_NAME = "out_Domain";
+    private static final String CONNECTING_LINE_REGISTERED_RESOURCE_ELEMENT_NAME = "connectingLine_RegisteredResource";
+    private static final String MEASUREMENT_UNIT_ELEMENT_NAME = "measurement_Unit.name";
+    private static final String CURVE_TYPE_ELEMENT_NAME = "curveType";
+    private static final String PERIOD_ELEMENT_NAME = "Period";
+    private static final String RESOLUTION_ELEMENT_NAME = "resolution";
+    private static final String TIME_INTERVAL_ELEMENT_NAME = "timeInterval";
+    private static final String POINT_ELEMENT_NAME = "Point";
+    private static final String POSITION_ELEMENT_NAME = "position";
+    private static final String QUANTITY_ELEMENT_NAME = "quantity";
+    private static final String REASON_ELEMENT_NAME = "Reason";
+    private static final String CODE_ELEMENT_NAME = "code";
+    private static final String TEXT_ELEMENT_NAME = "text";
+
+    private PevfExchangesXml() {
+    }
+
+    public static void read(PevfExchanges pevfExchanges, XMLStreamReader reader) throws XMLStreamException {
```

Comment:
I prefer to return an instance of PevfExchanges instead of filling an existing one. Moreover, the constructor is private package so it's unusable outside this module